conans/test/util/tools_test.py

```diff
@@ -409,10 +410,16 @@ def vcvars_constrained_test(self):
         with self.assertRaisesRegexp(ConanException,
                                      "compiler.version setting required for vcvars not defined"):
             tools.vcvars_command(settings)
+
+        import requests
```

Comment:
Why necessary? requests shouldn't be used in this test at all? What happens if you pass None?